package/MDAnalysis/core/groups.py

```diff
@@ -3087,6 +3361,23 @@ def __repr__(self):
         return "{}, with selection{} {} on {}>".format(basestr[:-1],
                     "s"[len(self._selection_strings)==1:], sels, basegrp)
 
+    @property
+    def atoms(self):
+        """Get a *static* :class:`AtomGroup` identical to the group of currently
+        selected :class:`Atoms<Atom>` in the :class:`UpdatingAtomGroup`.
```

Comment:
AtomGroups being immutable, the change in behavior should not matter. It should be documented, though.

Reply:
@orbeckst @jbarnoud is right.  There's nothing you can do to equal (containing same atoms) AtomGroups that would make them different from each other.  Therefore it's not important if it returns a copy or self

Reply:
IMHO it's quite important that `UpdatingAtomGroup.atoms` returns an `AtomGroup` instead of `self`. Otherwise, there'd be no convenient way of comparing the UAG's contents between different trajectory frames.

Reply:
Yes, that's fine with me (and a good justification).

As I said, I am happy with all the decisions, just waiting for the tiny doc updates.